hoomd_organics/base/system.py

```diff
@@ -248,7 +360,7 @@ def _remove_hydrogens(self):
             self.gmso_system.remove_site(site=h)
 
     def _scale_charges(self):
-        """"""
+        """Scale charges to net zero."""
```

Comment:
We could expand a bit and say something like, if the net charge isn't zero after applying for the forcefield, this will equally shift negative charges and positive charges across all particles to reach a net charge of zero.